geopandas/io/tests/test_parquet.py

```diff
@@ -52,10 +60,11 @@ def test_validate_dataframe():
         _validate_dataframe("not a dataframe")
 
 
+@pytest.mark.skipif(not HAS_PYARROW, reason="requires pyarrow")
```

Comment:
I *think* you can put this once before the first test, just as a statement and not as decorator, and it should skip all tests in here